Sources/ArcGISToolkit/Components/Form/FormInputs/DateTimeEntry.swift

```diff
@@ -202,6 +204,13 @@ struct DateTimeEntry: View {
     }
 }
 
+private extension DateFormatter {
+    static let arcGISDateFormatter: DateFormatter = {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss"
+        return dateFormatter
+    }()
+}
```

Comment:
Apple's [Data Formatting](https://developer.apple.com/documentation/foundation/data_formatting) document recommends only using formatters in Objective-C. In Swift, there are (mostly) equivalent formatting APIs that don't require creating a formatter directly. I have found one case where I could only do what I wanted using a formatter, but that is a rare exception. In this case, I think you could use [`Date.init(_:strategy:)`](https://developer.apple.com/documentation/foundation/date/3798897-init) to do the same thing without using a formatter.

Using the new formatter APIs involves a change of mindset. Before those new APIs came along, creating formatters to format data was ubiquitous. Now, we have to train ourselves not to reach for a formatter when needing to format data and instead, search for the replacement API that does the same thing, but in a more Swift-y way.

Reply:
@philium Good point, thanks for the reminder!

Reply:
@philium I've not been able to find a usage of `Date.init(_:strategy:)` that'll work with the date strings we have.

The format of the dates we're parsing is: `1969-07-07T20:17:00`.

I've tried a number of combinations of modifiers on `Date.ISO8601FormatStyle`.  More specifically, I've not been able to find a suitable `Date.ISO8601FormatStyle.DateTimeSeparator` for the `T`.

Reply:
You need to create the formatter yourself using a fixed format:

```swift
let strategy: Date.ParseStrategy = .fixed(
    format: "\(year: .defaultDigits)-\(month: .defaultDigits)-\(day: .defaultDigits)T\(hour: .defaultDigits(clock: .twentyFourHour, hourCycle: .zeroBased)):\(minute: .defaultDigits):\(second: .defaultDigits)",
    timeZone: .current
)
try Date("1969-07-07T20:17:00", strategy: strategy) // 1969-07-08 03:17:00 +0000
```